src/test/kotlin/three/consulting/epoc/repository/CustomerServiceIntegrationTest.kt

```diff
@@ -27,4 +29,23 @@ class CustomerServiceIntegrationTest : IntegrationTest() {
         val customer: CustomerDTO? = customerService.findCustomerForId(1000L)
         assertThat(customer).isNull()
     }
+
+    @Test
+    fun `added customer is found from database`() {
+        val customerName = "Älykäs Oy"
+        val customerDescription = "Innovating new innovation"
+        val customer = CustomerDTO(null,customerName, customerDescription, enabled = false)
```

Comment:
For readability's sake you can just create the customer dto object like
```
val customer = CustomerDTO(
    name = "Älykäs Oy",
    description = "Innovating new innovation"
)
```
All the other fields have default values so they can be ignored as long as you specify the other parameter names. Then you can also verify that if you don't touch the `enabled` field that it's created as `true` by default.